mod_reforged/msu_systems/tooltips.nut

```diff
@@ -25,7 +25,7 @@
 	},
 	Tactical = {
 		Button = {
-			WaitTurnAllButton = ::MSU.Class.BasicTooltip("Wait Round", "Have all your characters use \'Wait\' on their turn.")
+			WaitTurnAllButton = ::MSU.Class.BasicTooltip("Wait Round (Alt+R)", "Have all your characters use \'Wait\' on their turn.")
```

Comment:
Is it possible that the keybind mentioned in the brackets is dynamically pulled from what the user has set the keybind to?

Reply:
I only know that this is in line with the other vanilla tooltips (which obviously didnt expect to ever be changed)

Reply:
Try using `::MSU.Class.CustomTooltip`. Like shown below. Test if that works for fetching the keybind dynamically and if yes then let's go with that.

::MSU.Class.CustomTooltip(function( _ ) {
	return [
		{
			id = 1,
			type = "title",
			text = "Wait Round " + GETKEYBINDHERE
		},
		{
			id = 2,
			type = "description",
			text = DESCRIPTIONHERE
		}
	];
});

Reply:
I tried that out and it works.
Only ugly thing about is is that MSU does not offer an interface for accessing the keybind combination, or the keybind object in the first place.
So I had to do that in a manual way

Reply:
@TaroEld @Enduriel We need to add a way in MSU to get a keybind object by id.

Reply:
@Darxo Could you please push the commit with your current manual approach?

Reply:
Done